docs/cn_omf.md

```diff
@@ -203,16 +214,92 @@ This is followed by the bloom filter.
 
 ### HyperLogLog
 
-This region does not have a separate header, just the hyperloglog (hlog) bytes.  The offset and length in the KBlock header points to this region.  Each kblock stores its hlog bytes and the kvset's hlog can be composed using the individual kblock hlogs.
+This region does not have a separate header, just the hyperloglog (hlog) bytes.
+The offset and length in the KBlock header points to this region. Each kblock
+stores its hlog bytes and the kvset's hlog can be composed using the individual
+kblock hlogs.
 
 ## VBlocks (Value blocks)
 
-VBlocks consist of a header followed by all the values.  The offset of a specifc value and its length are obtained from the kmd entry for this value which is linked to the corresponding key.
-
-### VBlock header
+VBlocks consist of a header followed by all the values. The offset of a specifc
+value and its length are obtained from the kmd entry for this value which is
+linked to the corresponding key.
+
+### VBlock trailer
+
+    +---------------------+
+    | u32 vbh_magic       |
+    | u32 vbh_version     |
+    | u64 vbh_vgroup      |
+    | u32 vbh_min_key_off |
+    | u32 vbh_min_key_len |
+    | u32 vbh_max_key_off |
+    | u32 vbh_max_key_len |
+    |---------------------|
+    | min key             | 1344 bytes
+    |---------------------|
+    | max key             | 1344 bytes
+    +---------------------+
```

Comment:
We talked about getting rid of vgroup here too.  I don't think we need it.

A description of the meaning of min/max key is essential because storing min/max keys in vblocks seem unusual.